sapi/cli/php_cli_server.c

```diff
@@ -241,23 +241,23 @@ static const char php_cli_server_css[] = "<style>\n" \
 /* }}} */
 
 #ifdef PHP_WIN32
-int php_cli_server_get_system_time(char *buf) {
+static bool php_cli_server_get_system_time(char *buf) {
```

Comment:
I don't this isn't your code, but the definition could be shared for all architectures.